content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -286,7 +286,7 @@ StatefulSet will then begin to recreate the Pods using the reverted template.
 
 ### Minimum ready seconds
 
-{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
 
 `.spec.minReadySeconds` is an optional field that specifies the minimum number of seconds for which a newly
```

Comment:
It's worth mentioning that starting from 1.23 it's on by default

Reply:
We should reword line 296. maybe bring it right under this shortcode:

> Please note that this field only works if you enable the `StatefulSetMinReadySeconds` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).